modules/ROOT/pages/release-notes.adoc

```diff
@@ -24,6 +24,13 @@ The recommended approach to perform operations on a bucket dedicated to Couchbas
 - {url-issues-sync}/3740[*#3740*] Persist temporary revision body backups on import
 - {url-issues-sync}/3756[*#3756*] DCP Feed does not seed from all nodes
 
+*Known Issues*
+
+- {url-issues-sync}[*N/A*] Index Panic issue: the indexer throws an `index out of range` runtime error.
+* *Workaround:* disable xref:2.1@index.adoc#guaranteed-availability[GSI indexing].
```

Comment:
Might be better to phrase this as 'Query failures when querying xattrs with non-covering index', as there are multiple underlying query service issues fixed in 5.5.3:

https://issues.couchbase.com/browse/MB-31990
https://issues.couchbase.com/browse/MB-32120